[WIP] general: flash window when match is about to start

mittermichal opened the pull request:
using extension system ported from ETe

Commit:
general: flash window when match is about to start

using extension system ported from ETe

mittermichal commented:
@ensiform :wave: am I doing this right?

Commit:
cgame: verify if extension system is available

mittermichal commented:
acording to ensiform its all bad, i am adding WIP to title

ensiform commented:
https://gist.github.com/ensiform/014869907c5e821960399c21c25b43f5

Untested but should work.

ensiform committed:
general: correctly use extension system to flash window

patched by Ensiform, commit message by mittermichal

patched by Ensiform, commit message by mittermichal

Commit:
Merge branch 'flash-window-ensiform' into window-flash

# Conflicts:
#	src/cgame/cg_local.h
#	src/cgame/cg_servercmds.c
#	src/cgame/cg_syscalls.c
#	src/client/cl_cgame.c
